test/tests/functional/pbs_acct_log.py

```diff
@@ -107,18 +93,6 @@ def test_long_resource_end(self):
         log_match = 'resources_used.foo_str=' + hstr
         self.server.log_match("%s;.*%s.*" % (jid, log_match), regexp=True)
 
-        mailpass = 0
```

Comment:
If you are not going to look in the mail any more, you don't need to set ATTR_m to 'e' above.
Also, if you want to, you could change the for loop which creates hstr to hstr = '1' * 4096.  It'd probably be faster.

Reply:
Thanks, @bhroam for your review comments,
I have addressed your comments, please review further.